src/pybind/mgr/volumes/fs/operations/versions/subvolume_v1.py

```diff
@@ -774,7 +774,7 @@ def get_pending_clones(self, snapname):
                     raise VolumeException(-e.args[0], e.args[1])
                 else:
                     try:
-                        # If clone is completed between 'list_all_keys_with_specified_values_from_section'
+                        # If clone is completed between 'filter_keys'
                         # and readlink(track_id_path) call then readlink will fail with error ENOENT (2)
                         # Hence we double check whether track_id is exist in .meta file or not.
                         # Edge case scenario.
```

Comment:
nit: we don't following coding style so closely but ideally we need  to re-wrap this comment.